src/adapter/src/coord/command_handler.rs

```diff
@@ -117,15 +116,14 @@ impl Coordinator {
                     self.handle_privileged_cancel(conn_id);
                 }
 
-                Command::AppendWebhook {
+                Command::GetWebhook {
                     database,
                     schema,
                     name,
                     conn_id,
-                    received_at,
                     tx,
                 } => {
-                    self.handle_append_webhook(database, schema, name, conn_id, received_at, tx);
+                    self.handle_get_webhook(database, schema, name, conn_id, tx);
```

Comment:
(I'm aware this isn't part of your PR. But if you are changing this message, let's improve it more.)

Having the connection id here logically doesn't make sense. `resolve_entry` requires it which is why it's here in the first place. But the connection id during name resolution is only used to find temp objects. Webhooks cannot be temporary and are never associated with a connection. And the response to the `GetWebhook` command should never depend on a connection id. `for_sessionless_user` uses a connid of `SYSTEM_CONN_ID`, and that should probably be hard coded here? I'm a bit worried about rbac issues if anything sees that as the connection id and then allows for reading something that shouldn't be readable? But...webhooks are always externally available so that's not real. I'm pretty sure the conn id has nothing to do with rbac.

Reply:
Good callout, I fixed this and hard coded `SYSTEM_CONN_ID`